### ISSUE TITLE
"indent" rule blames the wrong line if initial line is indented badly

### ISSUE THREAD

**Issue by mohsen1** [eslint/eslint#3933]
This is the code that I had:

```js
  'use strict'; // <-- line 1 (badly indented)

import {compose} from 'yaml-js';
import {load, dump} from 'js-yaml';
import _ from 'lodash';
```

The warnings were pointing to any line but the one that was indented badly:
```
    3:1   warning  Expected indentation of 2 space characters but found 0  indent
    4:1   warning  Expected indentation of 2 space characters but found 0  indent
    5:1   warning  Expected indentation of 2 space characters but found 0  indent
    7:1   warning  Expected indentation of 2 space characters but found 0  indent
    9:1   warning  Expected indentation of 2 space characters but found 0  indent
   10:1   warning  Expected indentation of 2 space characters but found 0  indent
   11:1   warning  Expected indentation of 2 space characters but found 0  indent
   12:1   warning  Expected indentation of 2 space characters but found 0  indent
   13:1   warning  Expected indentation of 2 space characters but found 0  indent
   14:1   warning  Expected indentation of 2 space characters but found 0  indent
   16:1   warning  Expected indentation of 2 space characters but found 0  indent
   17:1   warning  Expected indentation of 2 space characters but found 0  indent
   20:1   warning  Expected indentation of 2 space characters but found 0  indent
```

**Comment by eslintbot** [comment]
Thanks for the issue! If you're reporting a bug, please be sure to include:

1. The version of ESLint you are using (run `eslint -v`)
2. What you did (the source code and ESLint configuration)
3. The actual ESLint output complete with numbers
4. What you expected to happen instead

Requesting a new rule? Please see [Proposing a New Rule](http://eslint.org/docs/developer-guide/contributing/new-rules) for instructions.

**Comment by gyandeeps** [comment]
duplicate https://github.com/eslint/eslint/issues/3832
